src/app.ts

```diff
@@ -34,6 +103,23 @@ console.log('***********************')
 // @ts-expect-error
 global.clog = console.log
 
+// TODO: Remove isMounted from IGNORED_WARNINGS once we upgrade to RN 0.57
+const IGNORED_WARNINGS = [
+  'Module RCTSplashScreen requires main queue setup since it overrides `constantsToExport`',
+  /Warning: \w+ has been renamed, and is not recommended for use. See [^\s]* for details./,
+  'slowlog',
+  'Setting a timer for a long period of time',
+  'Warning: isMounted(...) is deprecated'
+]
+// $FlowExpectedError
+// @ts-expect-error
+console.ignoredYellowBox = IGNORED_WARNINGS
+
+// Ignore errors and warnings(used for device testing)
+if (ENV.DISABLE_WARNINGS) {
+  LogBox.ignoreAllLogs() //  (IGNORED_WARNINGS)
+}
```

Comment:
We shouldn't be filtering warnings because they motivate us to handle the tech-debt rather than bury it. I'm okay with a DISABLE_WARNINGS ENV flag if developers want to use this, but it still discourages actually handling the warnings, so it's not great.

Reply:
Sorry this entire file wasn't supposed to end up in this review